source/winVersion.py

```diff
@@ -49,12 +50,23 @@ def _windowsVersionToReleaseName(self):
 		elif (self.major, self.minor) == (6, 3):
 			return "Windows 8.1"
 		elif self.major == 10:
-			buildsToReleases = {build: release for release, build in WIN10_RELEASE_NAME_TO_BUILDS.items()}

```

Comment:
If this is the only usage, and it reverses the mapping, you could consider adding a new mapping, deprecating the old one. The new mapping would be internal only, and map builds to release names. Though this isn't necessary if we just construct the `class WinVersion` with the release name populated.